presto-main/src/main/java/com/facebook/presto/translator/registry/ConstantTypeRegistry.java

```diff
@@ -0,0 +1,34 @@
+package com.facebook.presto.translator.registry;
+
+import com.facebook.presto.spi.type.Type;
+import com.facebook.presto.spi.type.TypeSignature;
+import com.google.common.collect.ImmutableSet;
+
+import java.util.Arrays;
+import java.util.Set;
+
+import static com.facebook.presto.translator.registry.ScalarFromAnnotationsParser.removeTypeParameters;
+
+public class ConstantTypeRegistry
```

Comment:
For registering constant types. May consider combining Constant and Function registries. Lmk if a better way to do this.